data/net/minecraft/world/phys/shapes/BooleanOp.mapping

```diff
@@ -0,0 +1,52 @@
+CLASS net/minecraft/world/phys/shapes/BooleanOp
```

Comment:
`Bool` is not a word. I'd call all of these `left` and `right` as semantically that's what they are in a two input function (left hand side <op> right hand side => result)